lmnet/cpu.requirements.txt

```diff
@@ -1,2 +1,3 @@
 -r dev.requirements.txt
-tensorflow==1.13.1
+tensorflow==1.14.0
+gast==0.2.2
```

Comment:
Why tensorflow automatically solve dependency of `gast`...
Anyway LGTM.

Reply:
I guess TF 1.14.0 just uses the latest gast, because TF people doesn't expected the incompatible update of `gast`.